app/include/dt-bindings/zmk/kscan-mock.h

```diff
@@ -4,11 +4,4 @@
  * SPDX-License-Identifier: MIT
  */
 
-#pragma once
-
-#define ZMK_MOCK_IS_PRESS(v) ((v & (0x01 << 31)) != 0)
-#define ZMK_MOCK_PRESS(row, col, msec) (row + (col << 8) + (msec << 16) + (0x01 << 31))
-#define ZMK_MOCK_RELEASE(row, col, msec) (row + (col << 8) + (msec << 16))
-#define ZMK_MOCK_ROW(v) (v & 0xFF)
-#define ZMK_MOCK_COL(v) ((v >> 8) & 0xFF)
-#define ZMK_MOCK_MSEC(v) ((v & ~(0x01 << 31)) >> 16)
+#include "kscan_mock.h"
```

Comment:
Can we add a warning above this: https://stackoverflow.com/a/3826868

Reply:
Something like this?
`#warning "'kscan-mock.h' has been deprecated and replaced by 'kscan_mock.h'"`

Reply:
>  along with a warning/deprecation notice
apologies for missing that btw

Reply:
Done, tested here:
- https://github.com/innovaker/zmk/actions/runs/438462011
- https://github.com/innovaker/zmk/actions/runs/438462009